ocs_ci/ocs/ui/views.py

```diff
@@ -1324,13 +1324,30 @@
     "name-input-btn": ("#name-input", By.CSS_SELECTOR),
     "vm-page-next-btn": ("button[type='submit']", By.CSS_SELECTOR),
     "dr-policy": ("#pf-select-toggle-id-0", By.CSS_SELECTOR),
-    "select-policy": ("(//span[@class='pf-v5-c-select__menu-item-main'])[1]", By.XPATH),
+    "select-dr-policy": (
+        "(//span[@class='pf-v5-c-select__menu-item-main'])[1]",
+        By.XPATH,
+    ),
     "selected-protection-type": ("//div[normalize-space()='{}']", By.XPATH),
     "assign": ("button[type='submit']", By.CSS_SELECTOR),
     "conf-msg": ("(//h4[@class='pf-v5-c-alert__title'])[1]", By.XPATH),
     "close-page": ("button[aria-label='Close']", By.CSS_SELECTOR),
     "select-shared": ("#shared-vm-protection", By.CSS_SELECTOR),
     "select-drpc": ("input[name='radioGroup']", By.CSS_SELECTOR),
+    "dr-status": ('span[data-test="status-text"]', By.CSS_SELECTOR),
+    "popover_text": ("//div[@data-test-id='popover-header']", By.XPATH),
+    "kebab-action": (
+        "(//td[@class='pf-v5-c-table__td pf-v5-c-table__action']//button)[2]",
```

Comment:
pls don't use locators with indexes, by reasons already mentioned in another comment. best option is to use text of action item

Reply:
Only if the element has visible text on page it will contain text.

On your choice you can use @area-label combining it with @type="button".
Also, did you try existing locator bellow?
```
"kebab-action": (
        "//button[contains(@class, 'c-dropdown__toggle pf-m-plain')]",
        By.XPATH,
    ),
```
https://github.com/red-hat-storage/ocs-ci/blob/be08fda025e880ba4d3cbb6e5990432eb111576f/ocs_ci/ocs/ui/views.py#L1126-L1129